lib/charms/prometheus_k8s/v0/prometheus.py

```diff
@@ -713,7 +713,7 @@ def __init__(
         service_event,
         jobs=[],
         multi=False,
-        alert_rules_path="prometheus_alert_rules",
+        alert_rules_path="src/prometheus_alert_rules",
```

Comment:
Ideally should be `os.path.join` but I am ambivalent because I am not sure charms will ever run on any other OS other than a Linux.

Reply:
Precisely